packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -202,7 +207,10 @@ contract Bridge is EssentialContract, IBridge {
         uint256 refundAmount;
 
         // Process message differently based on the target address
-        if (message.to == address(this) || message.to == address(0)) {
+        if (
+            message.to == address(0) || message.to == address(this)
+                || message.to == ethVault || message.to == resolve("taiko", true)
```

Comment:
Is this only added here for messages that were already sent before adding the new checks? Just want to make sure I don't miss anything.

Reply:
Without this, we'd have to rely on all source AddressManagers to be configured correctly; with this, the Bridge is considered more secure.